eclipse/sw360
# feat(ProjectUI): Disabled CR based on project Group

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.

- To enable / disable the `clearing request` feature for `projects` based on project `business unit` (Group).

Issue: https://github.com/eclipse/sw360/issues/1434

### Suggest Reviewer
> You can suggest reviewers here with an @mention.

### How To Test?

- Set the `Business Unit` (Group) for which the `Clearing Request` should be disabled in properties file (add comma separated `Group` for multiple values).
- property key: `org.eclipse.sw360.disable.clearing.request.for.project.group`
- `CR` should be disabled only for those project which belonged to `Business Unit` which are mentioned in above properties file.
- User should not be able to create the new `CR` for projects belonging to `Business Unit` which are mentioned in properties file.


### Checklist
Must:

Signed-off-by: Abdul Kapti <abdul.kapti@siemens-healthineers.com>
